frontend/pages/gamescreen/styles.css

```
@@ -227,3 +296,65 @@ hr {
+.leaflet-popup-content-wrapper{
+        background-color: rgba(0,0,0,0.5);
+    color: white;
+    border: 1px solid rgba(255,255,100,0.4);
+    backdrop-filter: blur(10px);
+    border-radius: 5px;
+    font-family: poppins , sans-serif;
+
+}
+.leaflet-popup-content{
+
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)